templates/layoutAdmin.html.twig

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html.twig" %}
+{% extends "baseAdmin.html.twig" %}
```

Comment:
hum...  non en fait ton layoutAadmin doit etendre base et non un baseAdmin
comme ça ton layout visiteur et layout admin on une base commune, c'est plu DRY

Reply:
ma baseAdmin est différente car elle permet l'affichage de la nav sur le côté et une inclusion du body ce que ne permet la base normal

Reply:
Ton BaseAdmin = ton LayoutAdmin.
tu reprends base et ce que tu as mis dans baseAdmin.html.twig tu le mets directement dans layoutAdmin.

Reply:
Je ne peux pas le faire j'avais déjà essayé...
Le layout admin ne peut pas étendre un bloc body vide, il y a un message d'erreur, il faut qu'il y ait un fichier twig include dedans, or on ne peut pas inclure un fichier body de toutes les pages admin
J'ai besoin d'avoir un container avec un structure de 2 col et 10 col et des div
je ne peux pas le remettre dans le fichier base.html.twig car ça va casser la structure du site..

Reply:
Je ne peux pas faire ça si c'est l'idée que vous avez:  
{% extends "base.html.twig" %}
<div class="container-fluid">
    <div class="row">
     {% block header %}{% include '_admin_nav.html.twig' %}{% endblock %}
        <div class="col-10">
            {% block body %}{% endblock %}
        </div>
    </div>
</div>

Reply:
si, il faut que ton base soit indépendant de la mise en page. 
le layout visiteur impose to container et tes row d'une sorte
le layout admin la forme 2/10 que tu veux.
Il te faut un seul fichier base commun qui déclare le doctype html, sinon tu n'as aucun intéret d'avoir un fichier base ;-)

Reply:
On n'a pas réussi, on a passé quelques temps dessus avec Anthony hier, je te montrerai le pb, j'ai supprimé la base Admin mais la solution carrément nulle est de laisser une div ouverte : le row de la nav bar n'est pas fermé
car même si je mets une col de 2 à ma nav bar le body lui prend toute la page ( = même body que sur le site utilisateur) et du coup il se met en dessous et pas côte à côte. Je ne peux mettre mettre de div dans le layout admin ni de block body vide avec des div autour de col-10, symfony renvoit des messages d'erreur. On regardera ensemble comment solutionner ça